Pipfile

```diff
@@ -21,8 +21,9 @@ sqlalchemy = "*"
 maya = "*"
 flask = "*"
 # Third-Party Ethereum
-py-evm = "*"
+py-evm = "==0.2.0a39"
```

Comment:
:+1: This a very recent version.

Reply:
I had some problems with `py-evm==0.2.0a41`

Reply:
Not a blocker, but what Issues did you get?

Reply:
...
```
eth_tester/backends/pyevm/main.py", line 63, in <module>
    from eth.vm.spoof import (
ModuleNotFoundError: No module named 'eth.vm.spoof'
```